README.md

```diff
@@ -20,6 +20,9 @@ Der aus Projekt I entstandende [Prototyp](https://www.figma.com/proto/Jbh4haqz3F
 ## Dokumentation
 Eine detaillierte Dokumentation der einzelnen Projektergebnisse, Entscheidungen und Standards findest du unter [/docs](./docs/README.md).
 
+## Haftungsausschluss
+Hiermit wird bekannt gegeben, dass die Stakeholder*innen und Entwickler*innen dieser Komponente keine Verantwortung für die folgenden Fälle übernehmen:<br>
```

Comment:
Das klingt sehr juristisch, Herr Noss meinte ja, dass wir lieber in normaler Sprache darauf hinweisen sollten, dass Images herunterladbar sind. Was meint ihr? So lassen oder umschreiben?